src/relayclient/GsnTestEnvironment.ts

```diff
@@ -145,20 +145,22 @@ class GsnTestEnvironmentClass {
      */
     const interactor = new ContractInteractor(new Web3.providers.HttpProvider(host),
       configureGSN({ relayHubAddress: deploymentResult.relayHubAddress }))
-    const relayServerParams = {
+    const relayServerDependencies = {
       contractInteractor: interactor,
       txStoreManager,
       managerKeyManager,
-      workersKeyManager,
+      workersKeyManager
+    }
+    const relayServerParams = {
       url: relayUrl,
       hubAddress: deploymentResult.relayHubAddress,
       gasPriceFactor: 1,
-      baseRelayFee: 0,
+      baseRelayFee: '0',
```

Comment:
Maybe keep it consistent with the rest i.e. make all integers strings or keep them all as integers?